pallets/staking-rewards/src/migration.rs

```diff
@@ -67,8 +69,10 @@ impl<T: Config> OnRuntimeUpgrade for MigrateToV2<T> {
     fn post_upgrade(state: Vec<u8>) -> Result<(), &'static str> {
         use parity_scale_codec::Decode;
 
-        let inflation: Perquintill = Decode::decode(&mut &state[..]).unwrap();
-        assert_eq!(inflation, Perquintill::from_percent(6),);
+        let old_inflation: Perquintill = Decode::decode(&mut &state[..]).unwrap();
+        let new_inflation = pallet::TargetInflation::<T>::get();
+        assert_ne!(old_inflation, new_inflation);
```

Comment:
no asserts

Reply:
It's a tests that runs locally even without any chain state